src/Export.php

```diff
@@ -141,10 +141,10 @@ private function updateSelectFields(array $currentITem, $selectFields)
         foreach ($selectFields as $selectFieldDefinitionKey => $selectFieldDefinition) {
             $selectFieldData = $currentITem['fields'][$selectFieldDefinitionKey];
             $data = $this->populateSelectFieldReferencedData($selectFieldData, $selectFieldDefinitionKey, $selectFieldDefinition);
-        }
 
-        // Update the reference of the imported select field value.
-        $currentITem['fields'][$selectFieldDefinitionKey] = $data;
+            // Update the reference of the imported select field value.
+            $currentITem['fields'][$selectFieldDefinitionKey] = $data;
```

Comment:
Let's fix the capitalisation of this (and also in the method definition, on line 139) 

```suggestion
            $currentItem['fields'][$selectFieldDefinitionKey] = $data;
```

Reply:
Good catch @bobdenotter. 

It's fixed in https://github.com/bobdenotter/conimex/pull/28/commits/c625c7fc3d783819e984017c218ea92c7d78b65d